imports/pages/devotionals/devotions.Single.js

```diff
@@ -102,6 +102,16 @@ export default class SeriesSingle extends Component {
     this.props.dispatch(headerActions.hide());
   }
 
+  shouldComponentUpdate(nextProps, nextState) {
+    if (this.state.selectedIndex !== nextState.selectedIndex) return true;
+    if (nextProps.devotion.content && this.props.devotion.content) {
+      if (nextProps.devotion.content.id === this.props.devotion.content.id) return false;
+      this.setState({ selectedIndex: 0 });
```

Comment:
@edolyne this fixes the problem. I agree this solution isn't ideal, but I think the real solution involves dropping react-swipe-views, as it's not being maintained and there is a better plugin now.